librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -28,7 +28,7 @@
 **********************************************************************/
 
 // Changes: https://github.com/LibreCAD/LibreCAD/commits/master/librecad/src/main/qc_applicationwindow.cpp
-
+#include <iostream>
```

Comment:
The _iostream_ header may be removed. I was using this header while performing tests.